test/e2e/specs/editor/blocks/spacer.spec.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Spacer', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can be created by typing "/spacer"', async ( { editor, page } ) => {
+		// Create a spacer with the slash block shortcut.
+		await page.click( 'role=button[name="Add default block"i]' );
+		await page.keyboard.type( '/spacer' );
+		await page.keyboard.press( 'Enter' );
+
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'can be resized using the drag handle and remains selected after being dragged', async ( {
+		page,
+		editor,
+	} ) => {
+		// Create a spacer with the slash block shortcut.
+		await page.click( 'role=button[name="Add default block"i]' );
+		await page.keyboard.type( '/spacer' );
+		await page.keyboard.press( 'Enter' );
+
+		const resizableHandle = page.locator(
+			'[aria-label="Block: Spacer"] .block-library-spacer__resize-container .components-resizable-box__handle'
```

Comment:
The resize handle is not accessible by design, but we could still add a `data-testid` attribute to help target it in tests. For now, we should add a comment above explaining why we're using a class name selector.

Also, we can chain role selectors with CSS selectors using `>>`:

```suggestion
			'role=document[name="Block: Spacer"i] >> css=.components-resizable-box__handle'
```